src/routes/estimations/estimations.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Body, Controller, HttpCode, Param, Post } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { EstimationRequest } from './entities/estimation-request.entity';
+import { EstimationResponse } from './entities/estimation-response.entity';
+import { EstimationsService } from './estimations.service';
+
+@ApiTags('estimations')
+@Controller({
+  path: '',
+  version: '2',
+})
+export class EstimationsController {
+  constructor(private readonly estimationsService: EstimationsService) {}
+
+  @ApiOkResponse({ type: EstimationResponse })
+  @HttpCode(200)
```

Comment:
Why are we using the `@HttpCode` here?

Reply:
Because the current Rust implementation returns a 200 status code, and being a POST method Nest would return 201 by default.